### PR TITLE
Add missing ASL (license) to GitHub Actions

### PR DESCRIPTION
**Pull request by vorburger**
@cushon merge this? Just a minor clean-up, to make #1048 shorter and easier to review.

### CODE REVIEW

**Comment by vorburger** [comment]
This will look "cleaner" to review after a rebase if you are willing to merge #1049, first.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Error Prone Authors.
+# Copyright 2020-2024 The Google Java Format Authors
```
Comment (cushon) [inline .github/workflows/ci.yml:1]: This is a nit, but I think the guidance is to use the year the file was created, and that it doesn't have to be updated to the current year, so just '2020' should be sufficient.

**Review by cushon — CHANGES_REQUESTED** [review]
Thanks, looks good aside from maybe adjusting the years (in both files).

File: .github/workflows/ci.yml (hunk 1 above)
Comment (vorburger) [reply .github/workflows/ci.yml]: sure thing, done!

**Pull request merged**